src/gentropy/dataset/l2g_features/vep.py

```diff
@@ -110,9 +110,11 @@ def common_neighbourhood_vep_feature_logic(
         local_metric.join(regional_max_per_study_locus, "studyLocusId", "left")
```

Comment:
Shouldn't we add here: 
gene_index.df.filter(f.col("biotype") == "protein_coding").select("geneId"),
too?